cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -179,6 +182,7 @@ public void deleteSchedule(Id.Program program, SchedulableProgramType programTyp
       scheduler.deleteSchedule(program, programType, scheduleName);
     } catch (NotFoundException e) {
       LOG.trace("Could not delete schedule", e);
+      Throwables.propagate(e);
```

Comment:
This method catches, logs and propagates exceptions, but the following method does not. Why?